apps/crn-server/src/validation/research-output.validation.ts

```diff
@@ -180,6 +180,14 @@ const researchOutputPostRequestValidationSchema: JSONSchemaType<ResearchOutputPo
         nullable: true,
         default: false,
       },
+      statusChangedById: {
+        type: 'string',
+        nullable: true,
+      },
+      hasStatusChanged: {
+        type: 'boolean',
+        nullable: true,
+      },
```

Comment:
I don't think this is necessary, right? we're only going to send `statusChangedById` and `hasStatusChanged` when it's a put request :)